src/components/nabla/Pools/Backstop/WithdrawLiquidity/useWithdrawLiquidity.ts

```diff
@@ -103,19 +171,17 @@ export const useWithdrawLiquidity = (pool: BackstopPool) => {
   );
 
   return {
-    address,
-    amount,
+    form,
+    amountString,
     backstopWithdraw,
+    backstopDrain,
     balanceQuery,
     depositQuery,
-    form,
-    isSwapPoolWithdraw,
-    pools,
     selectedPool,
-    swapPoolWithdraw,
-    tokenModal,
+    tokenToReceive,
+    withdrawalQuote,
     toggle,
+    onSubmit: form.handleSubmit(selectedPool !== undefined ? backstopDrain.onSubmit : backstopWithdraw.onSubmit),
```

Comment:
I wanted to confirm whether we are only focusing on `undefined` value and not considering `null` value in this check. Is that correct?

Reply:
`selectedPool` can never be `null`, only `undefined`, also ensured by TypeScript.